### PR TITLE
add_annotations_to_table_helper.py bugfixes

### PR DESCRIPTION
**Pull request by susannasiebert**
This addresses https://github.com/genome/cancer-genomics-workflow/issues/195 and https://github.com/genome/cancer-genomics-workflow/issues/191.

### CODE REVIEW

File: add_annotations_to_table_helper.py [hunk 1]
```diff
@@ -104,7 +109,9 @@ def transcript_for_alt(transcripts, alt):
             for alt in entry['ALT'].split(','):
                 vep_annotations = vep[entry['CHROM']][entry['POS']][entry['REF']][alt]
                 if vep_annotations is not None and field in vep_annotations:
-                    field_annotations.append(vep_annotations[field])
+                    annotation = vep_annotations[field]
+                    decoded_annotation = re.sub(r'%[0-9|A-Z][0-9|A-Z]', decode_hex, annotation)
```
Comment (tmooney) [inline add_annotations_to_table_helper.py]: If this is decoding hex, should it it only match on 0-9 and A-F?

File: add_annotations_to_table_helper.py [hunk 2]
```diff
@@ -16,16 +17,48 @@ def parse_csq_header(vcf_file):
...
         transcript = {}
         for key, value in zip(csq_fields, values):
             transcript[key] = value
-        transcripts.append(transcript)
+        if transcript['Allele'] not in transcripts.keys():
+            transcripts[transcript['Allele']] = []
+        transcripts[transcript['Allele']].append(transcript)
+    return transcripts
+
+def resolves_alleles(entry, csq_alleles):
```
Comment (tmooney) [inline add_annotations_to_table_helper.py]: `resolve_alleles` would match the verb forms of the other function names more closely.

**Review by tmooney — APPROVED** [review]
+1 ⛵️

File: add_annotations_to_table_helper.py (hunk 1 above)
Comment (susannasiebert) [reply add_annotations_to_table_helper.py]: Fixed.

File: add_annotations_to_table_helper.py (hunk 2 above)
Comment (susannasiebert) [reply add_annotations_to_table_helper.py]: I changed the method name as suggested.

**Pull request merged**